sync3/extensions/account_data.go

```diff
@@ -54,6 +57,13 @@ func (r *AccountDataRequest) AppendLive(ctx context.Context, res *Response, extC
 		}
 		// if this is a room update which is included in the response, send account data for this room
 		if _, exists := extCtx.RoomIDToTimeline[update.RoomID()]; exists {
+			// ..but only if we haven't before
+			if res.AccountData != nil && res.AccountData.loadedRooms[update.RoomID()] {
+				// we've loaded this room before, don't do it again
+				// this can happen when we consume lots of items in the buffer. If many of them are room updates
+				// for the same room, we could send dupe room account data if we didn't do this check.
+				break
```

Comment:
Does it make sense to just `return` here? (same for the other break above)
Given the control flow, we would be hitting the below
```go
if len(globalMsgs) == 0 && len(roomToMsgs) == 0 {
	return
}
```

Reply:
Oh, right you are! That will be clearer, thank you!